src/recipes/heartbeat-by-user-first-impression/index.js

```diff
@@ -161,11 +150,11 @@ let run = function (state, extras) {
   eData.store();
 
   let locale = (state.locale || "UNK").toLowerCase();
-  let trans = translations[locale] || {};
+  let trans = translations[locale] || translations['en-us']; //
 
-  let question_text = trans.question_text || "Please rate Firefox";
-  let learnmore = trans.learnmore || "Learn more";
-  let thankyou = trans.thankyou || "Thank you!";
+  let question_text = trans.question_text();
```

Comment:
Don't see a test to guarantee that these exist.  Should we add a test or a "|| <default>"?